bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/Layer/LayerTools.jsx

```diff
@@ -88,10 +97,17 @@ const LayerStatus = ({ backendStatus, model, onClick }) => {
         />
     );
 
-    return <Tooltip title={<Message messageKey={backendStatus.messageKey} />}>{icon}</Tooltip>;
+    const statusTitle = (
+        <Fragment>
+            { layerType && <Message messageKey={`layerTooltipTitle.${layerType}` } /> } <Message messageKey={backendStatus.messageKey} />
+        </Fragment>
```

Comment:
What about if we would pass something like "tooltipMessageKey={backendStatus.messageKey}" to LayerIcon itself? Then we could let LayerIcon handle the tooltip and it's formatting?

Reply:
Though LayerIcon is used on the selected layers listing as well: https://github.com/oskariorg/oskari-frontend/blob/0fed5dad3ce8106f7769cdc5cbf24cf2746dd67d/bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox/Footer/Footer.jsx#L59